testing/correctness/tests/resilience/resilience.py

```diff
@@ -360,15 +361,21 @@ def _test_resilience(command, ops=[], initial=None, sources=1,
 # Test Runner
 #############
 
+VALIDATION_CMD = 'validator -i {out_file} -a'
+EXPECT_STUB = ' -e {expect}'
 def _run(persistent_data, res_ops, command, ops=[], initial=None, sources=1,
-         partition_multiplier=1, validate_output=True,
+         partition_multiplier=1, validation_cmd=None,
          sender_mps=1000, sender_interval=0.01):
     host = '127.0.0.1'
     sinks = 1
     sink_mode = 'framed'
     batch_size = int(sender_mps * sender_interval)
     logging.debug("batch_size is {}".format(batch_size))
 
+    # If validation_cmd is False, it remains False and no validation is run
+    if validation_cmd is None:
```

Comment:
Would it be better to give `validation_cmd` a default value of `VALIDATION_CMD` instead of doing this? Is there a reason we want to give people the option of specifying `NONE` at the call site?

Reply:
@nisanharamati ^^

Reply:
Just to follow up, this is a question for @nisanharamati, but if this is a change we want, let's open a separate issue for it.

Reply:
I think @aturley 's suggestion is good.
Specifiying `None` or not specifying anything for the parameter will result in the default command being used...
Users _can_ specify `False` at the call site in cases where validation shouldn't be run at all (some long-running resilience tests, for example).

Reply:
Yes to @aturley 's suggestion.